client/js/analysisui/layoutcell.js

```diff
@@ -65,9 +72,9 @@ var LayoutCell = Backbone.View.extend({
 
         if (this.$previousContent !== null)
             this.trigger('layoutcell.contentChanged', Math.random());
```

Comment:
i did have that thought. I was trying to get around the issue that the id is to track layout events that can come from anywhere within the grid tree and therefore the id counter would need to be global to all grid objects within that tree and this was the easiest way although not perfect (if its not unique its not the end of the world, but it would be most of the time). I could use the prototype of the grid class to access some global id counter system. Or just pick a random number. That's the thought pattern anyway. I'll have a look at changing it.